frontend/src/App.js

```diff
@@ -50,6 +51,13 @@ function App() {
         {hasRole(currentUser, "ROLE_ADMIN") && (
           <Route exact path="/admin/requests" element={<AdminRequestsPage />} />
         )}
+        {/* {hasRole(currentUser, "ROLE_PROFESSOR") && (
+          <Route
+            exact
+            path="/requests/pending"
+            element={<PendingRequestsPage />}
+          />
+        )} */}
```

Comment:
can remove commented out code in App.js - it is implemented in https://github.com/ucsb-cs156-f24/proj-rec-f24-06/pull/88